docs/contributing/source/dev-setup-coding-and-contribution-process/run-node-with-docker.md

```diff
@@ -32,10 +32,10 @@ docker run \
   --env BIGCHAINDB_DATABASE_HOST=172.17.0.1 \
   bigchaindb/bigchaindb \
   -y configure \
-  [mongodb|rethinkdb]
+  localmongodb
```

Comment:
Correct me if I am wrong but I don't think `localmongodb` is needed when `configure`ing with `-y`, as the default backend is set to `localmongodb`

Reply:
yes not necessary. Just kept it for clarity.